docs/authentication/session-management-overview.mdx

```diff
@@ -0,0 +1,132 @@
+slug: /session-management-overview
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/08/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
+
+Session management is a crucial aspect of web application development. An authentication session is a period of user interaction with the application, typically starting with the user providing credentials and ending with logout or inactivity timeout. Effective session management ensures user data privacy, maintains session state, and provides a seamless user experience.
+
+The Beyond Identity session consists of a `token`, specifically an [access token](/docs/next/api-token-overview), which is implemented as a JSON Web Token (JWT). An access token, also used for API authenitcation, enables access to an application or resource. The token is digitally signed and contains information about what resources can be accessed and for how long.
+
+By default, a token expires after 24 hours, but this can be configured in the [Admin Console](/docs/next/session-management-console).
+
+Additionally, when you request an [access token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.4), it is possible to receive an additional token called a [refresh token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.5) in the response. The refresh token can be used to request additional access tokens with identical or narrower scope.
+
+## How to add session management to my application
+
+There are a few functions of tokens that you should be aware of.
+
+- [Create a Token](/docs/next/create-api-token) (Pre-Requisite)
+- [Store Tokens](#how-to-store-tokens)
+- [Create Session Cookie](#where-to-store-tokens)
+- [Introspect a Token](/docs/next/validate-access-tokens)
+- [Revoke a Token](/docs/next/revoke-access-tokens)
+- [Retrieve Active Tokens](/docs/next/list-access-tokens)
+
+If you have enabled refresh tokens for your application, you can also refresh the access token to extend the duration of the user's session without requiring authentication.
+
+- [Refresh token request](/docs/next/enable-refresh-tokens)
+
+## How to store tokens
+
+Applications may utilize cookies, local storage or other strategies to associate session with the end-user’s application. Beyond Identity doesn’t provide an API/SDK for this purpose; it is up to you to decide how to handle the end-user’s session. One option is to store the JWT received from Beyond Identity in the end user’s application. Another solution is to store the JWT received from Beyond Identity on the server and use a cookie. This is a common solution offered by many web frameworks and is more secure as the JWT is more sensitive.
```

Comment:
Did you have other web frameworks you could link the user to here? Or provide some guidance on libraries they can use?

Reply:
That is a good question. I did some research yesterday, and I mostly found references to built-in framework solutions ([OWASP](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html#built-in-session-management-implementations) names a few) or competitor solutions. I can add something though